app/views/posts/index.html.erb

```diff
@@ -1,26 +1,20 @@
-<h1>Listing Posts</h1>
- 
- <%= link_to 'New post', new_post_path %>
-<table>
-  <tr>
-    <th>Title</th>
-    <th>Text</th>
-    <th colspan="3"></th>
-  </tr>
- 
+<div>
+  <h1>Listing Posts</h1>
+  <%= paginate(@posts) %>
   <% @posts.each do |post| %>
-    <tr>
-      <td><%= post.title %></td>
-      <td><%= post.text %></td>
-      <td><%= link_to 'Show', post_path(post) %></td>
+  <div>
+    <p><%= post.title %></p>
+    <p><%= post.text %></p>
+      <%= link_to 'Show', post_path(post),class: "btn btn-primary "%>
       <%if user_signed_in?%>
-        <%if post.user_id == current_user.id%>
-          <td><%= link_to 'Edit', edit_post_path(post) %></td>
-          <td><%= link_to 'Destroy', post_path(post),
+      <%if post.user_id == current_user.id%>
+        <%= link_to 'Edit', edit_post_path(post),class: "btn btn-success "%>
```

Comment:
Please combine both if into sing one like this

if user_signed_in? &&  post.user_id == current_user.id